core/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.6-alpine
 MAINTAINER Abhilash Raj
 
 #Add startup script to container
-ADD assets/run.sh /opt/run.sh
+COPY docker-entrypoint.sh /usr/local/bin/
```

Comment:
I guess the semantics are same (in this case) for ADD and COPY so we shouldn't have any problem, I was just wondering if there is any reason to make this change.

We can stick with COPY here, but I would still make entrypoint script executable explicitly.